Performance/Benchmarks/Benchmarks/GRPCSwiftBenchmark/Benchmarks.swift

```diff
@@ -14,24 +14,119 @@
  * limitations under the License.
  */
 import Benchmark
-import Foundation
+import GRPCCore
 
 let benchmarks = {
-    Benchmark.defaultConfiguration = .init(
-        metrics: [
-            .mallocCountTotal,
-            .syscalls,
-            .readSyscalls,
-            .writeSyscalls,
-            .memoryLeaked,
-            .retainCount,
-            .releaseCount,
-        ]
-    )
-    
-    // async code is currently still quite flaky in the number of retain/release it does so we don't measure them today
-    var configWithoutRetainRelease = Benchmark.defaultConfiguration
-    configWithoutRetainRelease.metrics.removeAll(where: { $0 == .retainCount || $0 == .releaseCount })
-    
-    // Add Benchmarks here
+  Benchmark.defaultConfiguration = .init(
+      metrics: [
+          .mallocCountTotal,
+          .syscalls,
+          .readSyscalls,
+          .writeSyscalls,
+          .memoryLeaked,
+          .retainCount,
+          .releaseCount,
+      ]
+  )
+  
+  // async code is currently still quite flaky in the number of retain/release it does so we don't measure them today
+  var configWithoutRetainRelease = Benchmark.defaultConfiguration
+  configWithoutRetainRelease.metrics.removeAll(where: { $0 == .retainCount || $0 == .releaseCount })
+  
+  Benchmark("Metadata_Add_string") { benchmark in
+    for _ in benchmark.scaledIterations {
+      var metadata = Metadata()
+      for i in 0..<1000 {
+          metadata.addString("\(i)", forKey: "\(i)")
+      }
+    }
+  }
+  
+  Benchmark("Metadata_Add_binary") { benchmark in
+    let value: [UInt8] = [1, 2, 3]
+    for _ in benchmark.scaledIterations {
+      var metadata = Metadata()
+      
+      benchmark.startMeasurement()
+      for i in 0..<1000 {
+          metadata.addBinary(value, forKey: "\(i)")
+      }
+      benchmark.stopMeasurement()
+    }
+  }
+  
+  Benchmark("Metadata_Remove_values_for_key") { benchmark in
```

Comment:
Seems like we only need underscores in the filename:
- https://github.com/apple/swift-certificates/blob/main/Benchmarks/Benchmarks/CertificatesBenchmark/Benchmarks.swift
- https://github.com/apple/swift-certificates/tree/main/Benchmarks/Thresholds/5.9

Can we remove them here?

Reply:
This is what I had originally (files with underscores, benchmark names with spaces) but the check against the thresholds would always pass even when it shouldn't - I could only make it fail once I started using underscores in the names too.